chad6/homework#6_1.js

```diff
@@ -0,0 +1,70 @@
+// homework #6 part 1a
+
+function Ctor() {
+  return {
+    a: 0,
+    b: 1
+  }
+}
+
```

Comment:
This is definitely the right idea, but looks like Dan wanted you to use "this" in your answer, like so:

function Ctor() {
	this.a = 0;
	this.b = 1;
}